draft-ietf-quic-multipath.md

```diff
@@ -558,70 +565,47 @@ cannot be issued more than once on the same connection
-requires available connection IDs for the same Path ID on both sides. For instance,
-if the maximum Path ID limit is 2 and the endpoint wants to provide connection IDs
-for only one Path ID inside range \[1, 2\], it should select Path ID 1 (and not Path
-ID 2). Similarly, endpoints SHOULD consume Path IDs in a continuous way, i.e., when
-creating paths. However, endpoints cannot expect to receive new connection IDs
-or path initiation attempts with in order use of Path IDs
-due to out-of-order delivery or path validation failure.
-
 Endpoints MUST NOT issue new connection IDs with Path IDs greater than
 the Maximum Path Identifier field in MAX_PATH_ID frames (see {{max-paths-frame}})
 or the value of initial_max_path_id transport parameter if no MAX_PATH_ID frame was received yet.
 Receipt of a frame with a greater Path ID is a connection error as specified
-in {{frames}}.
+in Section {{frames}}.
+
 When an endpoint finds it has not enough available unused path identifiers,
 it SHOULD either send a MAX_PATH_ID frame to increase the active path limit
 (when limited by the sender) or a PATHS_BLOCKED frame
-(see Section {{paths-and-cids-blocked-frame}}) to inform the peer that a new path
-identifier was needed but the current limit set by the peer prevented the
-creation of the new path.
+(see Section {{paths-and-cids-blocked-frame}}) to inform the peer that
+its current limit prevented the creation of the new path.
 
 ### Rotating and Retiring Connection IDs
 
 {{Section 5.1.2 of QUIC-TRANSPORT}} indicates that an endpoint
 can change the connection ID it uses to another available one
 at any time during the connection. For the extension specified in
-this draft, endpoints MUST only rotate to another connection IDs associated
+this document, endpoints MUST only rotate to another connection IDs associated
 to the same Path ID. Use of a connection ID associated with
 another Path ID will be considered as an attempt to open a new path instead.
 
-{{Section 5.1.2. of QUIC-TRANSPORT}} specifies the retirement of connection IDs.
-In order to identify a connection ID correctly when the multipath extension is used,
-endpoints have to use the PATH_RETIRE_CONNECTION_ID frame instead
-of the RETIRE_CONNECTION_ID frame to indicate the respective Path ID together with the
-connection ID sequence number, at least for all paths with a Path ID other than 0.
-Endpoints can also use PATH_NEW_CONNECTION_ID and
-PATH_RETIRE_CONNECTION_ID for the initial path with Path ID 0,
-however, the use of NEW_CONNECTION_ID and RETIRE_CONNECTION_ID
-is still valid as well and endpoints need to process these frames accordingly
-as corresponding to Path ID 0.
+An endpoints is supposed to retire connection ID that are not used anymore,
+and the server is supposed to provide
+replacements, as specified in {{Section 5.1.2. of QUIC-TRANSPORT}}.
+As such when receiving a PATH_RETIRE_CONNECTION_ID frame, an endpoint
```

Comment:
```suggestion
As such, when receiving a PATH_RETIRE_CONNECTION_ID frame, an endpoint
```

Reply:
Might be better to just remove "As such,"